src/SerialPorts.jl

```diff
@@ -28,24 +28,8 @@ immutable SerialPort <: IO
 end
 
 function __init__()
-    try
-        copy!(PySerial, pyimport("serial"))
-        copy!(PySerialListPorts, pyimport("serial.tools.list_ports"))
-    catch e
-        if PyCall.conda
-            info("Installing serial via the Conda package...")
-            Conda.add("pyserial")
-            copy!(PySerial, pyimport("serial"))
-            copy!(PySerialListPorts, pyimport("serial.tools.list_ports"))
-        else
-            error("""Failed to pyimport("serial"): SerialPorts will not work until you have a functioning pyserial module.
-                  For automated serial installation, try configuring SerialPorts to use the Conda Python distribution within Julia.  Relaunch Julia and run:
-                        ENV["PYTHON"]=""
-                        Pkg.build("Serial")
-                        using PyPlot
-                  pyimport exception was: """, e)
-        end
-    end
+    copy!(PySerial, pyimport_conda("serial", "pyserial"))
+    copy!(PySerialListPorts, pyimport_conda("serial.tools.list_ports", "pyserial"))
```

Comment:
(You can just use `pyimport` on the second line, since `pyserial` should be installed by the time the first `pyimport_conda` completes.)